schema/mitwelten_v2.sql

```diff
@@ -201,6 +201,40 @@ CREATE TABLE IF NOT EXISTS prod.files_entry
     UNIQUE (object_name)
 );
 
+CREATE TABLE IF NOT EXISTS prod.image_results (
+  result_id serial PRIMARY KEY,
+  file_id int,
+  config_id varchar
+);
+
+CREATE TABLE IF NOT EXISTS prod.flowers (
+  flower_id serial PRIMARY KEY,
+  result_id int,
+  class varchar,
+  confidence float4,
+  x0 int,
+  y0 int,
+  x1 int,
+  y1 int
+);
+
+CREATE TABLE IF NOT EXISTS prod.pollinators (
+  pollinator_id serial PRIMARY KEY,
+  result_id int,
+  flower_id int,
+  class varchar,
+  confidence float4,
+  x0 int,
+  y0 int,
+  x1 int,
+  y1 int
+);
+
+CREATE TABLE IF NOT EXISTS prod.pollinator_inference_config (
+  config_id varchar PRIMARY KEY,
```

Comment:
Why `varchar` instead of `int`? Is this a requirement of prefect?

Reply:
No it is not. See the idea in the 1st conversation